doc/src/custom/auto_cli_doc.py

```diff
@@ -0,0 +1,485 @@
+#!/usr/bin/env python2
```

Comment:
This file can go as you have implemented another script to do the job.

Reply:
Yes, good spot; I forgot to remove this once I realised it was out of scope to adapt it to work for Cylc commands for this initial PR.